apps/ui/src/views/Space/Editor.vue

```diff
@@ -433,50 +458,73 @@ watchEffect(() => {
     <div class="flex items-stretch md:flex-row flex-col w-full md:h-full">
       <div class="flex-1 grow min-w-0">
         <UiContainer class="pt-5 !max-w-[710px] mx-0 md:mx-auto s-box">
-          <MessageVotingPower
-            v-if="propositionPower"
-            class="mb-4"
-            :voting-power="propositionPower"
-            action="propose"
-            @fetch-voting-power="handleFetchPropositionPower"
-          />
           <UiAlert
-            v-if="
-              propositionPower &&
-              spaceType === 'default' &&
-              proposalLimitReached
-            "
+            v-if="unsupportedProposalNetworks[0] && !proposal?.proposalId"
             type="error"
             class="mb-4"
           >
-            <span
-              >Please verify your space to publish more proposals.
-              <a
-                :href="VERIFIED_URL"
-                target="_blank"
-                class="text-rose-500 dark:text-neutral-100 font-semibold"
-                >Verify space</a
-              >.</span
-            >
+            <div>
+              You cannot create proposals. This space is configured with
+              <b>{{ unsupportedProposalNetworks[0].name }}</b
+              >, a non-premium network. Change to a
+              <AppLink
+                to="https://help.snapshot.box/en/articles/10478752-what-are-the-premium-networks"
+                >premium network
+                <IH-arrow-sm-right class="inline-block -rotate-45" />
+              </AppLink>
+              or upgrade <b>{{ unsupportedProposalNetworks[0].name }}</b> to
```

Comment:
Should we display more network names here?

We suggest that only one network is not premium, admins may update the settings for it and comeback to realize they need to update settings again